docs/source/callbacks.rst

```diff
@@ -45,6 +45,13 @@ Logging
 
 .. autoclass:: TensorBoardLogger
 
+Tracking
+--------
+
+.. autoclass:: WeightsGradientsStatsTracker
```

Comment:
I don't think this goes there. However, we should document and put the Tracker callback there IMO.

Reply:
Where should it go ?

Didn't document as similar approach as Logger.

Reply:
I'm not sure. Maybe we should not put it public at first.

Reply:
Ok with me. If we had more `WeightsGradientsStatsTracker `would be ok but it's more a intern class.

Reply:
Alright, then just let TensorBoardGradientTracker there.

Reply:
ebdf618